src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -42,7 +42,7 @@
 _MANAGED_BY_TENANTS = 'managedByTenants'
 _USER_ENTITY = 'user'
 _USER_NAME = 'name'
-_HOME_ACCOUNT_ID = 'homeAccountId'
+_USER_HOME_ACCOUNT_ID = 'homeAccountId'
```

Comment:
Prefix with `_USER` to sync with other entries like `_USER_NAME` and `_USER_TYPE`.